index.js

```diff
@@ -2577,6 +2579,8 @@
   //. Takes a unary predicate and a value of any type, and returns logical
   //. negation of applying predicate to the value.
   //.
+  //. See also (`not`)[#not]
```

Comment:
I think the reference is useful. I have a small request: please add a full stop (`.`) at the end of the line. :)